blog/nextflow.md

```diff
@@ -0,0 +1,712 @@
+---
+title: Basic Bioinformatics Workflows on OSCAR: Nextflow
+author: Jordan Lawson
+date: 2023-09-01
+slug: nextflow
+description: This blog discusses a workflow management tool, called **Nextflow**, its benefits, and how to use the software on Brown University's HPC cluster, OSCAR. 
+tags:
+  - 
+
+---
+<p><center>Creators: Ashok Ragavendran, Paul Cao, and Jordan Lawson</center>
+ <center>Center for Computation and Visualization</center>
+ <center>Center for Computational Biology of Human Disease - Computational Biology Core</center></p>
+
+Resources for help @brown <br> 
+
+COBRE CBHD Computational Biology Core
+- Office hours
+- https://cbc.brown.edu
+- slack channel on ccv-share
+- cbc-help@brown.edu <br>
+
+Center for Computation and Visualization
+- Office hours
+- https://ccv.brown.edu
+- ccv-share slack channel
+- https://docs.ccv.brown.edu
+- support@ccv.brown.edu
+
+
+# Overview 
+
+## What is Nextflow?  
+
+Nextflow is a workflow management tool that allow users to easily write data-intensive computational **pipelines**. These pipelines, or workflows as they are also called, have the following key features:
+
+- Sequential processing of files
+- Usually requires more than one tool
+- Multiple programming languages
+- Most times each sample is processed individually
+- Compute resource intensive 
+  - For example, in computational biology pipelines, there is often a process known as sequence alignment, which refers to the process of arranging the primary sequences of DNA, RNA, or protein to identify regions of similarity that are of interest 
+  - This alignment process could take 16 cpus, 60 Gb RAM, 4-24 hours, 30Gb of disk space per sample
+
+
+## Why do we care about these pipelines? 
+
+### Reason 1: Reproducibility 
+
+![image1](/content/images/blog/nextflow/reproduce.png) 
```

Comment:
for accessibility purposes, perhaps giving these images better alt text would be nice
```suggestion
![reproduce](/content/images/blog/nextflow/reproduce.png) 
```